### PR TITLE
Add organization management related identity configurations

### PR DESCRIPTION
**Pull request by sadilchamishka**
### Proposed changes in this pull request
$subject

### CODE REVIEW

**Review by dewniMW — APPROVED** [review]

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 1]
```diff
@@ -1873,6 +1873,31 @@
...
+        </Resource>
+        <Resource context="(.*)/api/server/v1/organizations/(.*)" secured="true" http-method="PUT">
+            <Permissions>/permission/admin/manage/identity/organizationmgt/update</Permissions>
+            <Scopes>internal_organization_update</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/server/v1/organizations/(.*)" secured="true" http-method="DELETE">
+            <Permissions>/permission/admin/manage/identity/organizationmgt/delete</Permissions>
+            <Scopes>internal_organization_delete</Scopes>
+        </Resource>
+
```
Comment (janakamarasena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:1900]: Lets remove the space between the configurations

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 2]
```diff
@@ -1873,6 +1873,31 @@
             <Scopes>internal_login</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/organizations" secured="true" http-method="GET">
```
Comment (janakamarasena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:1876]: Lets remove the empty new line between the configurations

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 3]
```diff
@@ -2557,6 +2557,31 @@
             <Scopes>internal_login</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/organizations" secured="true" http-method="GET">
```
Comment (janakamarasena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2560]: Lets remove the empty new line between the configurations

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 4]
```diff
@@ -2557,6 +2557,31 @@
...
+        </Resource>
+        <Resource context="(.*)/api/server/v1/organizations/(.*)" secured="true" http-method="PUT">
+            <Permissions>/permission/admin/manage/identity/organizationmgt/update</Permissions>
+            <Scopes>internal_organization_update</Scopes>
+        </Resource>
+        <Resource context="(.*)/api/server/v1/organizations/(.*)" secured="true" http-method="DELETE">
+            <Permissions>/permission/admin/manage/identity/organizationmgt/delete</Permissions>
+            <Scopes>internal_organization_delete</Scopes>
+        </Resource>
+
```
Comment (janakamarasena) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2584]: Lets remove the empty new line between the configurations

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml (hunk 2 above)
Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:1876]: The identity configurations to support organization management feature is removed from the identity.xml and indentity.xml.j2 files of the framework repository and added via the deployment.toml configuration at the B2B feature enable setup. (The isb2bconnector) https://github.com/wso2-enterprise/identity-org-mgt-connector/blob/main/modules/distribution/conf/deployment.toml

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml (hunk 1 above)
Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml:1900]: Related to the reply above https://github.com/wso2/carbon-identity-framework/pull/4074/files#r1004802482

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 3 above)
Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2560]: Related to the reply above https://github.com/wso2/carbon-identity-framework/pull/4074/files#r1004802482

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 (hunk 4 above)
Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2584]: Related to the reply above https://github.com/wso2/carbon-identity-framework/pull/4074/files#r1004802482

**Pull request merged**